### PR TITLE
Initial pull request

### PR DESCRIPTION
**Pull request by leowangx2013**
My first pull request

### CODE REVIEW

**Comment by Prakhar0409** [comment]
comment?

**Comment by xzhflying** [comment]
Please notice your PR/commit name and descriptions next time. @leowangx2013

File: .rspec [hunk 1]
```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```
Comment (xzhflying) [inline .rspec]: This file is not meant to be pushed...

Comment (leowangx2013) [inline .rspec]: ok

File: app/controllers/static_pages_controller.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class StaticPagesController < ApplicationController
+  def missionsystem_main
+  end
+
+  def missionsystem_help
+  end
+
+  def missionsystem_about
+  end
+end
```
Comment (xzhflying) [inline app/controllers/static_pages_controller.rb]: The static pages are handled by `high_voltage ` gem. Please learn how to use this gem.

File: app/views/layouts/application.html.slim [hunk 3]
```diff
@@ -1,33 +1,10 @@
...
-        = yield
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Ruby on Rails Tutorial Sample App | <%= yield(:title) %></title>
+  <%= stylesheet_link_tag    "application", media: "all",
+                                            "data-turbolinks-track" => true %>
+  <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
+  <%= csrf_meta_tags %>
+</head>
```
Comment (xzhflying) [inline app/views/layouts/application.html.slim]: please use slim syntax

File: app/views/static_pages/missionsystem_about.slim [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<% provide(:title, 'About Us') %>
+
+<h1>About Us</h1>
+<p>
+  The <a href="http://railstutorial.org/">Ruby on Rails Tutorial</a>
+  is a project to make a book and screencasts to teach web development
+  with <a href="http://rubyonrails.org/">Ruby on Rails</a>. This
+  is the sample application for the tutorial.
+</p>
```
Comment (xzhflying) [inline app/views/static_pages/missionsystem_about.slim]: slim syntax

File: app/views/static_pages/missionsystem_help.html.slim [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<% provide(:title, 'Help') %>
+
+<h1>Help</h1>
+<p>
+  Get help on the Ruby on Rails Tutorial at the
+  <a href="http://railstutorial.org/help">Rails Tutorial help page</a>.
+  To get help on this sample app, see the
+  <a href="http://railstutorial.org/book">Rails Tutorial book</a>.
+</p>
```
Comment (xzhflying) [inline app/views/static_pages/missionsystem_help.html.slim]: slim...

File: app/views/static_pages/missionsystem_main.html.slim [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<% provide(:title, 'Home') %>
+
+<h1>Sample App</h1>
+<p>
+  This is the home page for the
+  <a href="http://railstutorial.org/">Ruby on Rails Tutorial</a>
+  sample application.
+</p>
```
Comment (xzhflying) [inline app/views/static_pages/missionsystem_main.html.slim]: slim...

File: app/views/tasks/index.json.jbuilder [hunk 7]
```diff
@@ -0,0 +1,4 @@
+json.array!(@tasks) do |task|
+  json.extract! task, :id, :title, :content
+  json.url task_url(task, format: :json)
+end
```
Comment (xzhflying) [inline app/views/tasks/index.json.jbuilder]: what's this...

File: app/views/tasks/new.html.slim [hunk 8]
```diff
@@ -0,0 +1,5 @@
+h1 New task
+
+== render 'form'
```
Comment (xzhflying) [inline app/views/tasks/new.html.slim]: ==?

File: test/controllers/tasks_controller_test.rb [hunk 9]
```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
```
Comment (xzhflying) [inline test/controllers/tasks_controller_test.rb]: since you are using `rspec`, why do you still write your test in the `test` folder?

**Pull request closed without merging**